gems/smithy/lib/smithy/anvil/client/endpoint_parameter.rb

```diff
@@ -73,21 +83,14 @@ def client_context_doc
         # @return [value, source].  source may be one of [operation, config, default]
         def endpoint_parameter_value(operation)
           unless operation.nil?
-            source = 'operation'
             value = static_context_param(operation)
-            value ||= context_param_value(operation)
-            value ||= operation_context_param_value(operation)
+            value = context_param_value(operation) if value.nil?
```

Comment:
These changes are required to correctly handle "false" boolean values

Reply:
Makes sense